libc/src/time/ctime_s.h

```diff
@@ -0,0 +1,25 @@
+//===-- Implementation header of ctime_s ------------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LLVM_LIBC_SRC_TIME_CTIME_S_H
+#define LLVM_LIBC_SRC_TIME_CTIME_S_H
+
+#define __STDC_WANT_LIB_EXT1__ 1
```

Comment:
Do you need this define? This header is only included for the implementation in ctime_s.cpp, right?

```suggestion
```

Reply:
Ah, I wonder if this should be moved into, libc/hdr/types/rsize_t.h...let me play with that.